web/Dockerfile

```diff
@@ -20,8 +20,8 @@ RUN set -ex \
 		uwsgi \
 		psycopg2 \
 		dj-database-url \
-		pymysql \
-	&& pip install -U django==1.10 \
```

Comment:
I have seen before that if django is specified as a dependency for any of the components mentioned above, the version directive is ignored assuming that dependency is already satisfied.
To force installation of a specific version, I had to run it as a separate command here.